ios/FluentUI/People Picker/ContactCollectionViewLayout.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class ContactCollectionViewLayout: UICollectionViewFlowLayout {
+    @objc weak var delegate: UICollectionViewDelegateFlowLayout?
+
+    struct Constants {
+        static let itemWidth: CGFloat = 70.0
+        static let minLineSpacing: CGFloat = 14.0
+        static let maxLineSpacing: CGFloat = 16.0
+
+        static let extraSmallContentContactHeight: CGFloat = 115.0
+        static let smallContentContactHeight: CGFloat = 117.0
+        static let mediumContentContactHeight: CGFloat = 118.0
+        static let largeContentContactHeight: CGFloat = 121.0
+        static let extraLargeContentContactHeight: CGFloat = 125.0
+        static let extraExtraLargeContentContactHeight: CGFloat = 129.0
+        static let extraExtraExtraLargeContentContactHeight: CGFloat = 135.0
+    }
+
+    override init() {
+        super.init()
+        delegate = self
+        minimumLineSpacing = Constants.maxLineSpacing
+    }
+
+    required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) not implemented")
+    }
+
+    override var collectionViewContentSize: CGSize {
+        guard let collectionView = collectionView else {
+            return .zero
+        }
+
+        // TODO: Might not even need the setting here since the delegate method already sets it
+        if let minLineSpacing = delegate?.collectionView?(collectionView, layout: self, minimumLineSpacingForSectionAt: 0) {
+            minimumLineSpacing = minLineSpacing
+        }
+
+        let indexPath = IndexPath(item: 0, section: 0)
+        let numItems = collectionView.numberOfItems(inSection: 0)
+        let height = delegate?.collectionView?(collectionView, layout: self, sizeForItemAt: indexPath).height
+        let totalWidth = numItems > 0 ? CGFloat(numItems) * Constants.itemWidth + (CGFloat(numItems) - 1) * minimumLineSpacing : 0.0
+
+        let size = CGSize(width: totalWidth, height: height!)
+
+        return size
+    }
+
+    override func layoutAttributesForElements(in rect: CGRect) -> [UICollectionViewLayoutAttributes]? {
+        var layoutAttributes = [UICollectionViewLayoutAttributes]()
+
+        if let collectionView = collectionView {
+            numberOfItems = collectionView.numberOfItems(inSection: 0)
```

Comment:
numberOfItems shouldn't be a property, it's only used in this method, could make it a local var

Reply:
Good catch. Must have missed this when I was looking through it!